### DISCUSSION TITLE
Add Discord Rich Presence Support with Discord-Rpc package

### DISCUSSION THREAD

**Discussion by Nicat-dcw** [Acode-Foundation/Acode#854] [Ideas]
Add file name, language and others

**Comment by sebastianjnuwu** [comment]
Unfortunately it's not possible, I already tried to make a plugin about this but on mobile it's impossible to work using this lib or another similar one, and the rpc api is private

**Reply by Nicat-dcw** [reply]
Okay. Can you try with Discord User Tokens?
Set Presence with User Token

**Reply by Nicat-dcw** [reply]
@sebastianjnuwu can you create a plugin for it. It works with user tokens.you can use discord-self-bot-v13 package for made

**Reply by sebastianjnuwu** [reply]
If you could even do this, it would result in accounts getting banned from discord, is it worth the risk?

**Reply by Nicat-dcw** [reply]
Okay
